src/core/hash.js

```diff
@@ -17,7 +17,8 @@ let cache = {
  */
 export const hash = (compiled, target, glob) => {
     // generate hash
-    const className = cache[compiled] || (cache[compiled] = glob ? "" : toHash(compiled));
+    const comp_string = JSON.stringify(compiled);
```

Comment:
Great finding! 👍 

Would you mind changing the `const` name to camelCase? 😄

Reply:
Also, I think a test case for this would be helpful to not run into this issue, again.

Reply:
Oh ya I can do that. Been writing a bunch of python so used the wrong standard lol.

Reply:
Not exactly sure on how I should test it. I'll have a bit of time shortly to look into it and get a proper test. Maybe just it twice with 2 objects and ensure that the return is different?

Reply:
Fixed the casing and the references.

I updated the hash test at the end to do a check to ensure that returned vals are different for different objects. Let me know if it should change.